source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,228 @@
+===========
+Search Text
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you will learn how to run a **text search** in MongoDB.
+
+You can use a text search to retrieve documents that contain a **term**
+or a **phrase** in a specified field. A term is a sequence of characters
+that excludes whitespace characters. A phrase is a sequence of terms
+with any number of whitespace characters. 
+
+The following sections show you how to perform the following types of
+text searches:
+
+- Search Text by a Term
+- Search Text by a Phrase
+- Search Text with Terms Excluded
+
+If you want to sort your text search results, see the :ref:`Text Search
+<sorts-crud-text-search>` section of our Sort Results guide. 
+
+Sample Documents
+~~~~~~~~~~~~~~~~
+
+The following sections feature examples that run text searches on the
+``fast_and_furious_movies`` collection. Each section uses a variable
+named ``collection`` to refer to the ``MongoCollection`` instance of the
+``fast_and_furious_movies`` collection. 
+
+The collection contains documents with a title and tags associated
+with one of the several movies that are part of the Fast and Furious
+movie franchise: 
+
+.. code-block:: json
+
+   { "_id": 1, "title": "2 Fast 2 Furious ", "tags": ["undercover", "drug dealer"] } 
+   { "_id": 2, "title": "Fast 5", "tags": ["bank robbery", "full team"] }
+   { "_id": 3, "title": "Furious 7", "tags": ["emotional"] }
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+
+Text Index
+~~~~~~~~~~
+
+You must create a **text index** before running a text search. A text
+index specifies the string or string array field on which to run a text
+search. For more information, see the :ref:`Text Indexes <textIndexes>`
+section of our Indexes guide.
+
+In the following examples, you run text searches on the ``title``
+field in the ``fast_and_furious_movies`` collection. To enable text
+searches on the ``title`` field, create a text index using the
+:ref:`Indexes <index-text-indexes>` builder with the following 
+snippet: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin textIndex
+   :end-before: end textIndex
+
+Text Search
+-----------
+
+Use the ``Filters.text()`` method to specify a text search. 
+
+The ``Filters.text()`` method uses the :doc:`Filters builder
+</fundamentals/builders/filters>` to define a query filter specifying
+what to search for during the text search. You represent the query filter
```

Comment:
When "you represent", it generally means that you speak for someone or that something amounts to something. I think in this case, the query filter is represented by Bson rather than the reader speaking for the query filter or amounting to the query filter. Here's my previous suggestion:
"The query filter is represented by a Bson instance."
Link to the full suggestion: https://github.com/mongodb/docs-java/pull/85/files/47222d03539ffe913c8b47637e2a7fbcba7efcc1#r647027059